WeScan/ImageScannerController.swift

```diff
@@ -55,12 +55,21 @@ public final class ImageScannerController: UINavigationController {
         view.translatesAutoresizingMaskIntoConstraints = false
         return view
     }()
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: WeScan/ImageScannerController.swift;
  Line: 58;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)